.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+<!-- Provide a general summary of your changes in the Title above -->
+<!-- It must be meaningful and coherent with the changes -->
+
+<!--
+    If this PR is still a Work in Progress [WIP], please open it as DRAFT.
+    Please consider if any label should be added to this PR.
+-->
+
+## Description
+<!--
+    Describe changes in detail.
+    This includes depicting the context, use case or current behavior and describe the proposed changes.
+    If several features/bug fixes are included with these changes, please consider opening separated pull requests.
+-->
+
+<!--
+    In case of bug fixes, please provide the list of supported branches where this fix should be also merged.
+    Please uncomment following line, adjusting the corresponding target branches for the backport.
+-->
+<!-- @Mergifyio backport 1.4.x 1.2.x 1.0.x -->
+
+<!-- If an issue is already opened, please uncomment next line with the corresponding issue number. -->
+<!-- Fixes #(issue) -->
+
+<!-- In case the changes are built over a previous pull request, please uncomment next line. -->
+<!-- This PR depends on #(PR) and must be merged after that one. -->
+
+## Contributor Checklist
+
+- [ ] Commit messages follow the project guidelines. <!-- External contributors should sign the DCO. Fast DDS Python developers must also refer to the internal Redmine task. -->
+- [ ] Tests that thoroughly check the new feature have been added/Regression tests checking the bug and its fix have been added; the added tests pass locally
```

Comment:
Maybe we can split this one into two. What do you think about this?
```suggestion
- [ ] Tests that thoroughly check the new feature have been added /
      Regression tests checking the bug and its fix have been added
- [ ] The added tests pass locally
```

Reply:
I don't think I agree with this. In fact, I've just reused Fast DDS' template for the most part: https://github.com/eProsima/Fast-DDS/blob/master/.github/pull_request_template.md?plain=1#L34